Formula/easy-rsa.rb

```diff
@@ -1,33 +1,38 @@
 class EasyRsa < Formula
   desc "Homebrew formula for `easy-rsa`"
   homepage 'https://github.com/OpenVPN/easy-rsa'
-  url "https://github.com/OpenVPN/easy-rsa/archive/v3.0.3.tar.gz"
-  sha256 "3e41c0dfebe04332cdc07f13bf48a69e25dfc31d059e94071a9cfb9f3bba6f9a"
+  url "https://github.com/OpenVPN/easy-rsa/archive/v3.0.4.tar.gz"
+  sha256 "1d0da6cfc47b54565fe83068530ed467540879b853670257acb47d107e8f4df3"
   head "https://github.com/OpenVPN/easy-rsa.git"
 
   depends_on "openssl"
 
   devel do
-    version '3.0.3'
+    version '3.0.4'
   end
 
   def install
     inreplace "easyrsa3/vars.example",
-              "#set_var EASYRSA\t\"$PWD\"",
+              "#set_var EASYRSA\t\"${0%/*}\"",
+              "set_var EASYRSA \"#{Formula["easy-rsa"].opt_prefix}/share\""
+
+    inreplace "easyrsa3/easyrsa",
+              "set_var EASYRSA\t\t\"${0%/*}\"",
               "set_var EASYRSA \"#{Formula["easy-rsa"].opt_prefix}/share\""
 
     inreplace "easyrsa3/vars.example",
               "#set_var EASYRSA_OPENSSL\t\"openssl\"",
               "set_var EASYRSA_OPENSSL \"#{Formula["openssl"].opt_prefix}/bin/openssl\""
 
     inreplace "easyrsa3/easyrsa",
-              "\tlocal prog_vars=\"${0%/*}/vars\"",
-              "\tlocal prog_vars=\"#{Formula["easy-rsa"].opt_prefix}/share/vars\""
```

Comment:
@sovcharenko Thanks for your pull request. However, the goal is to use `vars` without setting after installed by `homebrew`
So please replace `prog_vars` to `prog_vars=\"#{Formula["easy-rsa"].opt_prefix}/share/vars`.

Reply:
Thanks for noticing! I made sensitive defaults inside easyrsa script itself, but forgot that someone may want to change them by placing their own vars into \"#{Formula["easy-rsa"].opt_prefix}/share/vars